index.html

```diff
@@ -55,6 +77,169 @@
 
     -->
 
-    <h1>Assignment 01 - To Be Completed</h1>
+    <!--Main page content-->
+    <!--Header-->
+    <header>
+      <h1>Kirby Fan Club</h1>
+      <img
```

Comment:
This works and there are no issues - but my general recommendation is to have your `h1` in the `main` area of your page.